pandas/tseries/tests/test_base.py

```diff
@@ -154,6 +155,86 @@ def test_minmax(self):
             obj = DatetimeIndex([pd.NaT, pd.NaT, pd.NaT])
             self.assertTrue(pd.isnull(getattr(obj, op)()))
 
+    def test_numpy_minmax(self):
+        dr = pd.date_range(start='2016-01-15', end='2016-01-20')
+        self.assertEqual(np.min(dr), Timestamp(
+            '2016-01-15 00:00:00', offset='D'))
+        self.assertEqual(np.max(dr), Timestamp(
+            '2016-01-20 00:00:00', offset='D'))
+
+        errmsg = "the 'out' parameter is not supported"
+        tm.assertRaisesRegexp(ValueError, errmsg, np.min, dr, out=0)
+        tm.assertRaisesRegexp(ValueError, errmsg, np.max, dr, out=0)
+
+        self.assertEqual(np.argmin(dr), 0)
+        self.assertEqual(np.argmax(dr), 5)
+
+        if not _np_version_under1p10:
+            errmsg = "the 'out' parameter is not supported"
+            tm.assertRaisesRegexp(ValueError, errmsg, np.argmin, dr, out=0)
+            tm.assertRaisesRegexp(ValueError, errmsg, np.argmax, dr, out=0)
+
+    def test_round(self):
```

Comment:
No, for two reasons:

1) `floor` and `ceil` are `ufunc` and are hence not defined in `fromnumeric.py`, pushing it outside the scope of this PR.

2) `floor` and `ceil` in `numpy` are used in a much different capacity than in `pandas` (IINM you are referring to `tslib.pyx`), making them inherently incompatible from the get-go.  Making them compatible in this instance would only encourage misuse of either IMO.